src/pages/guidelines/motion/overview.mdx

```diff
@@ -50,13 +50,13 @@ user’s attention and offer a rhythmic break to the productive experience.
 <Tab label="Curves">
 <Row>
 <Column colSm={6} colMd={4} colLg={6}>
-<p>Productive</p>
+<p className="tab--columns_label"> Productive </p>
 
 ![Curve graphs representing differences between productive and expressive animation curves](images/Motion_overview_Curves_productive.svg)
 
 </Column>
 <Column colSm={6} colMd={4} colLg={6}>
-<p>Expressive</p>
+<p className="tab--columns_label"> Expressive </p>
```

Comment:
```suggestion
<p className="tab--columns_label">Expressive</p>
```

is this intentional? I don't think this whitespace changes the text

Reply:
No, that wasn't intentional